### PR TITLE
Update version to 4.6.3

### PR DESCRIPTION
**Pull request by eakraly**
preparing for new tag 4.6.3

### CODE REVIEW

File: docker/coturn/Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ dockerify = $(strip $(if $(call eq,$(1),linux/arm32v6),linux/arm/v6,\
 # Project parameters #
 ######################
 
-COTURN_VER ?= 4.6.2
+COTURN_VER ?= 4.6.3
```
Comment (tyranron) [inline docker/coturn/Makefile]: For Docker image there is no point to change the version, since it's versioned a little bit different from the sources. It will be changed in a separate commit preparing the image release.

**Review by tyranron — APPROVED** [review]
@eakraly I've removed from changelog Docker-related stuff (and some unrelevant for end users CI stuff). Everything else seems LGTM.

File: docker/coturn/Makefile (hunk 1 above)
Comment (eakraly) [reply docker/coturn/Makefile]: Sure - just looks better when searching up the code and then seeing old version number is weird
